packages/rsbuild/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@nx/rsbuild",
+  "description": "The Nx Plugin for Rsbuild contains an Nx plugin, executors and utilities that support building applications using Rsbuild.",
+  "version": "0.0.1",
+  "type": "commonjs",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/nrwl/nx.git",
+    "directory": "packages/rsbuild"
+  },
+  "bugs": {
+    "url": "https://github.com/nrwl/nx/issues"
+  },
+  "keywords": [
+    "Monorepo",
+    "Rspack",
+    "Webpack",
+    "Bundling",
+    "Module Federation",
+    "Rsbuild"
+  ],
+  "author": "Colum Ferry",
+  "license": "MIT",
+  "homepage": "https://nx.dev",
+  "main": "index.js",
+  "executors": "./executors.json",
+  "dependencies": {
+    "tslib": "^2.3.0"
+  },
+  "peerDependencies": {},
+  "nx-migrations": {
+    "migrations": "./migrations.json"
+  }
+}
```

Comment:
Please add `"publishConfig": { "access": "public" }` to match other packages.

#29062 would catch this but I haven't had time to revisit the jest resolver issues yet

Reply:
done